.github/teams.yml

```diff
@@ -20,35 +14,34 @@ devrel:
   - '@kyleaton'        # Kyle Eaton
   - '@rdodev'          # Ruben Orduz
   - '@talagluck'       # Tal Gluck
+  - '@tjholsman'       # TJ Holsman
 
-cloud:
-  - '@roblim'          # Rob Lim
-  - '@rreinoldsc'      # Robby Reinold
-  - '@joshua-stauffer' # Josh Stauffer
-  - '@dctalbot'        # David Talbot
-  - '@wookasz'         # Łukasz Lempart
-  - '@josectobar'      # José Tobar
-  - '@elenajdanova'    # Elena Jdanova
+core:
   - '@DrewHoo'         # Drew Hoover
-  - '@lockettks'       # Kim Mathieu
-  - '@superengi'       # Saahir Foux
-
-# Aggregates a few different teams
-core-team:
-  # Mario
-  - '@NathanFarmer'    # Nathan Farmer
-  - '@alexsherstinsky' # Alex Sherstinsky
-  - '@cdkini'          # Chetan Kini
-  - '@billdirks'       # Bill Dirks
   - '@Kilo59'          # Gabriel Gore
-  # Luigi
+  - '@NathanFarmer'    # Nathan Farmer
   - '@Shinnnyshinshin' # Will Shin
-  - '@anthonyburdi'    # Anthony Burdi
-  - '@kenwade4'        # Ken Wade
-  # Misc
-  - '@abegong'         # Abe Gong
-  - '@jcampbell'       # James Campbell
-  - '@donaldheppner'   # Don Heppner
   - '@Super-Tanner'    # Tanner Beam
+  - '@abegong'         # Abe Gong
+  - '@alexsherstinsky' # Alex Sherstinsky
   - '@allensallinger'  # Allen Sallinger
+  - '@anthonyburdi'    # Anthony Burdi
+  - '@billdirks'       # Bill Dirks
+  - '@cdkini'          # Chetan Kini
+  - '@dctalbot'        # David Talbot
+  - '@donaldheppner'   # Don Heppner
+  - '@elenajdanova'    # Elena Jdanova
+  - '@jcampbell'       # James Campbell
+  - '@josectobar'      # José Tobar
+  - '@joshua-stauffer' # Josh Stauffer
+  - '@jshaikGX'        # Javed Shaik
+  - '@kenwade4'        # Ken Wade
+  - '@lockettks'       # Kim Mathieu
+  - '@roblim'          # Rob Lim
+  - '@rreinoldsc'      # Robby Reinold
   - '@sujensen'        # Susan Jensen
+  - '@tyler-hoffman'   # Tyler Hoffman
+  - '@wookasz'         # Łukasz Lempart
+
+bot:
+  - '@dependabot'      # Dependabot
```

Comment:
Does it need to be `@dependabot[bot]` like in the [release page](https://github.com/great-expectations/great_expectations/releases/tag/0.16.7)? Actually I see it both ways...

Reply:
hmm will update to include both